### PR TITLE
add trace clarification and version details.

### PR DESCRIPTION
**Pull request by drakirnosslin**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [ x ] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

### CODE REVIEW

File: site/content/_shared/variables-and-secrets.md [hunk 1]
```diff
@@ -7,4 +7,11 @@ There are two ways to store configuration information in Checkly: Variables and
 - **Variables** are used to store non-sensitive information. Variables are shown in plaintext when being edited, on the check result page and in logs. Variables can be accessed via the CLI and API.
 - **Secrets** allow you to store sensitive data for use in checks. Once saved secrets are never shown in the UI or in logs. The secret value cannot be accessed via the CLI or API.
 
+{{< warning >}}
```
Comment (miliberlin) [inline site/content/_shared/variables-and-secrets.md:10]: I think this section should also be added to `/docs/groups/variables/#environment-variables-and-secrets`. Wdyt?

File: site/content/_shared/variables-and-secrets.md [hunk 2]
```diff
@@ -7,4 +7,11 @@ There are two ways to store configuration information in Checkly: Variables and
...
+Note that for browser checks, secrets are only hidden in the Checkly native UI, not in any generated Playwright artifacts, such as Traces, Screenshots or Videos.
+{{< /warning >}}
+{{<info>}}
+Secrets are available for [Private Locations](/docs/private-locations/) on agent version `3.3.4` and later.
```
Comment (miliberlin) [inline site/content/_shared/variables-and-secrets.md:14]: Can you also add the CLI requirement here?

Comment (drakirnosslin) [reply site/content/_shared/variables-and-secrets.md]: Done.

File: site/content/docs/groups/variables.md [hunk 3]
```diff
@@ -12,6 +12,10 @@ You can set both variables and secrets for a Group of checks.
 
 For browser and multistep checks, you can set variables at the check level. See [browser check variables and secrets](/docs/browser-checks/variables) for more details.
 
+{{< warning >}}
+Note that for browser checks, secrets are only hidden in the Checkly native UI, not in any generated Playwright artifacts, such as Traces, Screenshots or Videos.
+{{< /warning >}}
+
```
Comment (miliberlin) [inline site/content/docs/groups/variables.md:18]: Let's add the requirements here too. Just to be safe 😅

```suggestion
{{<info>}}
Secrets are available for [Private Locations](/docs/private-locations/) on agent version `3.3.4` and later. Secrets are available on [CLI](/docs/cli/) version `4.9.0` and later.
{{</info>}}
```

**Review by miliberlin — APPROVED** [review]
LGTM

**Pull request merged**